<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTenses</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget
{
    color: #aaa;
   background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(54, 54, 54), stop:1 rgb(128, 128, 128));
}

</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>661</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
   color: rgb(255, 255, 255);
	background-color: #323232;
    border-radius: 20px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>In 1939 the Second World War ... .</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
   color: rgb(255, 255, 255);
	background-color: #323232;
    border-radius: 10px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Вопрос 1/5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton
{
   color: rgb(255, 255, 255);
	background-color: #323232;
    border-radius: 20px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>1 вариант ответа</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton
{
   color: rgb(255, 255, 255);
	background-color: #323232;
    border-radius: 20px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>2 вариант ответа</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton
{
   color: rgb(255, 255, 255);
	background-color: #323232;
    border-radius: 20px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>3 вариант ответа</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton
{
   color: rgb(255, 255, 255);
	background-color: #323232;
    border-radius: 20px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>4 вариант ответа</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    color: rgb(255, 255, 255);
	background-color: #323232;
    width: 75px;
    height: 50px;
	font-size: 30px;
	border-radius: 20px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}

QPushButton:hover
{
    background-color: silver;
}</string>
    </property>
    <property name="text">
     <string>Проверить</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>421</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit
{
   color: rgb(255, 255, 255);
	background-color: #323232;
    border-radius: 15px;
	font: &quot;Microsoft YaHei UI&quot;;
	font-weight: bold;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
